core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceAdminService.scala

```diff
@@ -98,23 +118,41 @@ extends LazyLogging
       }
     }
 
+  def getWorkspaceById(workspaceId: UUID): Future[WorkspaceAdminResponse] =
+    samDAO.admin
```

Comment:
Why not just put this as part of the `for` comprehension?
```suggestion
    for {
      userIsAdmin <- samDAO.admin.userHasAction(SamResourceTypeNames.workspace, SamResourceTypeAdminActions.readSummaryInformation, ctx)
      _ = if (!userIsAdmin) throw new RawlsExceptionWithErrorReport(
        ErrorReport(StatusCodes.Forbidden, "You must be an admin to call this API.")
      )
      workspaceOpt <- workspaceRepository.getWorkspace(workspaceId)
      workspace = workspaceOpt.getOrElse(throw NoSuchWorkspaceException(workspaceId.toString))
      settings <- workspaceSettingRepository.getWorkspaceSettings(workspaceId)
    } yield WorkspaceAdminResponse(
      WorkspaceDetails.fromWorkspaceAndOptions(workspace, None, useAttributes = false),
       settings
    )
```

Reply:
No compelling reason, I can switch it